src/app/utils/app-utils.ts

```diff
@@ -52,6 +53,14 @@ export class AppUtils {
     return stringCapitalised;
   }
 
+  public static formatDateAtTime(date: Date, is24Hour: boolean): string {
```

Comment:
As mentioned, can we create a custom pipe that extends angular's date pipe

Reply:
DOne